test/rhsmlib/dbus/test_config.py

```diff
@@ -10,64 +10,63 @@
 # Red Hat trademarks are not licensed under GPLv2. No permission is
 # granted to use or replicate Red Hat trademarks that are incorporated
 # in this software or its documentation.
+import tempfile
+from typing import Any, Dict
 
 import dbus
 
 from rhsm.config import RhsmConfigParser
-from rhsmlib.dbus import constants
 from rhsmlib.dbus.objects.config import ConfigDBusObject
-from test.rhsmlib.base import DBusObjectTest, TestUtilsMixin
+from test.rhsmlib.base import DBusServerStubProvider
 
 from test.rhsmlib.services.test_config import TEST_CONFIG
-from test import subman_marker_dbus
 
 
-@subman_marker_dbus
-class TestConfigDBusObject(DBusObjectTest, TestUtilsMixin):
-    def setUp(self):
-        super(TestConfigDBusObject, self).setUp()
-        self.proxy = self.proxy_for(ConfigDBusObject.default_dbus_path)
-        self.interface = dbus.Interface(self.proxy, constants.CONFIG_INTERFACE)
+class TestConfigDBusObject(DBusServerStubProvider):
+    dbus_class = ConfigDBusObject
+    dbus_class_kwargs: Dict[str, Any] = {"parser": None}
```

Comment:
I see that the type hints are written again here from the parent class, and the other attributes here do not have the types again; should the hint be removed here for consistency with the others?